treefmt.toml

```diff
@@ -0,0 +1,6 @@
+[formatter.nix]
+command = "nixpkgs-fmt"
+includes = ["*.nix"]
+excludes = [
```

Comment:
Probably want to exclude the auto-generated `pkgs/development/haskell-modules/hackage-packages.nix` as well, it causes the majority of the diff.

```
 pkgs/development/haskell-modules/hackage-packages.nix | 777268 ++++++++++++++++++++++++------------
 1 file changed, 524107 insertions(+), 253161 deletions(-)
```

Reply:
Thanks for this pointer. Added.